cves/kernel/CVE-2019-15807.yml

```diff
@@ -309,9 +320,9 @@ discussion:
 
     Put any links to disagreements you found in the notes section, or any other
     comment you want to make.
-  discussed_as_security:
-  any_discussion:
-  note:
+  discussed_as_security: false
+  any_discussion: false
+  note: 'There must have been some communication being done in-person.'
```

Comment:
This is just out of curiosity but why do you believe there must have been some communication being done in person?